c/pthread/bigshot_p_false-unreach-call.i

```diff
@@ -1213,11 +1213,13 @@ void __VERIFIER_assert(int expression) { if (!expression) { ERROR: __VERIFIER_er
 char *v;
 void *thread1(void * arg)
 {
-  v = malloc(sizeof(char) * 8);
+  v = calloc(8, sizeof(char));
```

Comment:
Why do we need `calloc` here? We never read the allocated memory, but potentially override it with new values.